packages/sync-actions/test/product-sync-variants.spec.js

```diff
@@ -982,6 +982,99 @@ describe('Actions', () => {
     ])
   })
 
+  test('update product-variant', async () => {
```

Comment:
I don't really understand this test. I thought the problem is that some array type properties aren't defined? Where can I see this in your test?

Reply:
i should be testing this func `moveMasterVariantsIntoVariants` but its not exported,
so i have to start with this func `SyncAction`.

On before object, `masterVariant` which is nested object from product, which contain these props 
`
       prices: [],
        images: [],
        attributes: [],
        assets: [],
`
while now object doesnt contain these props which causing the issue  
`TypeError: Cannot read property '0' of undefined`